client/components/sites-dropdown/style.scss

```diff
@@ -1,8 +1,9 @@
 @import "../../assets/stylesheets/shared/mixins/breakpoints";
 
 .sites-dropdown {
+	position: relative;
```

Comment:
The work in this file prevents the layout from shifting beneath the site dropdown when it's opened.